openapi.yaml

```diff
@@ -4711,16 +4711,32 @@ components:
       example: null
       description: |-
         The namespace the `process_id` is valid for.
+
+        The following options are pre-defined by the openEO API, but additional
+        namespaces may be introduced by back-ends or in a future version of the API.
         
-        By default, the namespace is `null`, which includes all pre-defined processes listed at `GET /processes`.
-        The namespace `user` refers to all user-defined processes listed as `GET /process_graphs`.
-        Additional namespaces may be introduced by back-ends or in a future version of the API.
+        * `null` (default): Checks both user-defined and pre-defined processes,
+           but prefers user-defined processes if both are available.
+           This allows users to add missing pre-defined processes for portability,
+           e.g. common processes from [processes.openeo.org](https://processes.openeo.org)
+           that have a process graph included.
+           If a user-defined process is prioritized over a pre-defined process,
+           a log entry SHOULD be added.
+        * `backend`: Uses exclusively the pre-defined processes listed at `GET /processes`.
+        * `user`: Uses exclusively the user-defined processes listed at `GET /process_graphs`.
+
+        If multiple processes with the same identifier exist, Clients SHOULD
+        inform the user that it's recommended to select a namespace.
     process_id:
       type: string
       description: |-
-        Unique identifier for the process.
+        An identifier for the process.
```

Comment:
> An

Maybe even better `The`, because `An` implies there are multiple identifiers for the same process (unless that is what you mean).

Reply:
Yes, will fix.